src/plexil-adapter/OwInterface.h

```diff
@@ -63,7 +64,7 @@ using FaultMap64 = std::map<std::string,std::pair<uint64_t, bool>>;
 class OwInterface
 {
  public:
-  static OwInterface* instance();
+  static std::shared_ptr<OwInterface> instance();
   OwInterface ();
   ~OwInterface ();
   OwInterface (const OwInterface&) = delete;
```

Comment:
If you take my advice on removing the deconstructor you can remove the copy and assignment constructors
```suggestion
```
and also remove next line 71: `OwInterface& operator= (const OwInterface&) = delete;`

Reply:
@Samahu I was not aware of any connection between the destructor and copy/assignment.  I've known these to be entirely orthogonal...

Reply:
https://en.wikipedia.org/wiki/Rule_of_three_(C%2B%2B_programming)

Reply:
A key point is that this class should remain a Singleton and so copy and assignment should be disallowed.  I think any adjustments here are outside the scope of this PR.  I'd say this change can be merged as is!

Reply:
I agree, since this is a singleton class we better keep these 'delete' statements. However, I don't think the comments were outside the scope of the PR. My objective was to make the code more concise given the new changes, some of them made sense and some not given the way OwInterface is implemented. Cheers!